docs/examples/obtain_refresh_token.py

```diff
@@ -39,17 +39,26 @@ def send_message(client, message):
 
 def main():
     """Provide the program's entry point when directly executed."""
-    if len(sys.argv) < 2:
-        print('Usage: {} SCOPE...'.format(sys.argv[0]))
-        return 1
-
-    reddit = praw.Reddit(client_id='YOUR_CLIENT_ID',
-                         client_secret='YOUR_CLIENT_SECRET',
+    print("Go here while logged into the account you want to create a token for: https://www.reddit.com/prefs/apps/")
```

Comment:
For consistency with the other PRAW code, can you use single quotes instead of double quotes around these strings? I believe if you run `flake8` on this file, it should provide you with some suggestions.